Emby.IsoMounting/IsoMounter/LinuxIsoManager.cs

```diff
@@ -87,9 +87,12 @@ public LinuxIsoManager(ILogger logger, IFileSystem fileSystem, IEnvironmentInfo
                 UmountCommand
             );
 
-            if (!string.IsNullOrEmpty(SudoCommand) && !string.IsNullOrEmpty(MountCommand) && !string.IsNullOrEmpty(UmountCommand)) {
+            if (!string.IsNullOrEmpty(SudoCommand) && !string.IsNullOrEmpty(MountCommand) && !string.IsNullOrEmpty(UmountCommand))
```

Comment:
is there any way to detect lines that are very long and cut them up a little? Here it would be very nice with
```
if(a
&& b
&& c)
```

Reply:
Not sure actually. Maybe ReSharper has it.

Reply:
Yes, resharper can do that. Out of the box it puts new lines at max line length.

Reply:
This will have to be done some other time. The PR is big enough as is.